CONTRIBUTING.md

```diff
@@ -16,3 +16,10 @@ You might be wondering what a contribution is. Here are some things that you can
 - Review [open Pull Requests](https://github.com/swapagarwal/swag-for-dev/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc).
 
 Thanks again, we hope to see you soon! ❤
+
+## How do I add a Swag Opportunity?
+
+1. Open an issue [here](https://github.com/swapagarwal/swag-for-dev/issues/new?template=----new-swag-opportunity.md)
+2. Fork the Repo
+3. Edit data.json to add your swag opportunity. Please make it in alphabetical order as well as using the data you first posted the issue about the swag opportunity
+4. Open a pull request!
```

Comment:
Mention that all the template has to be filled out (including original issue number)